src/components/Header.js

```diff
@@ -1,10 +1,64 @@
 import React from 'react';
 import styled from 'styled-components';
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
+export default function Header({ user, profileImg, changeUser }) {
+  const handleButtonClick = () => {
+    changeUser();
+  };
+  return (
+    <Wrapper>
+      <button onClick={handleButtonClick}>
+        <img src={profileImg} alt="userProfile"/>
+      </button>
+      <Stack>
+        <h2>{user}</h2>
+        <div>현재 활동중</div>
+      </Stack>
```

Comment:
Stack은 무슨 의미인가요?.? `UserInfo `같은 네이밍이 더 명확하고 좋을 것 같아요!
아래 제안한 코드는 예시일뿐이고,
재용님이 생각하시기에 더 괜찮은 네이밍이 있다면 그걸로 바꿔서 반영해주세요~ㅎㅎ

```suggestion
      <UserInfo>
        <UserName>{user}</UserName>
        <UserOn>현재 활동중</UserOn>
      </UserInfo>
```

Reply:
styled-components 사용방법을 이제 깨달았네요 감사합니다 ㅎㅎ